py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -2541,3 +2541,28 @@ def aten_ops_sort(
         dim=args_bounds_check(args, 1, -1),
         descending=args_bounds_check(args, 2, False),
     )
+
+
+@dynamo_tensorrt_converter(torch.ops.aten.copy.default)
+@enforce_tensor_types(
+    {
+        0: (TRTTensor,),
```

Comment:
Should this be `1: (TRTTensor,)`? We don't seem to use the first argument in the converter

Reply:
Thanks for the good catch!